package.json

```diff
@@ -1,9 +1,11 @@
 {
   "scripts": {
-    "postinstall": "lerna exec --concurrency=1 -- yarn && lerna exec --concurrency=99 -- linklocal --named @essex/timebrush ../timebrush"
+    "postinstall": "lerna exec --concurrency=1 -- yarn && lerna exec --concurrency=99 -- linklocal --named @essex/timebrush ../timebrush",
+    "test": "lerna --stream --scope @essex/timebrush run test && lerna run test --stream --ignore @essex/timebrush",
+    "test:powerbi": "lerna --stream --scope @essex/timebrush run test && lerna run test --stream --scope @essex/timebrush-powerbi"
   },
   "devDependencies": {
-    "lerna": "2.0.0-beta.34",
+    "lerna": "^2.0.0-beta.38",
     "linklocal": "^2.8.0"
```

Comment:
I can, but how does that solve the `linkLocal` issue?  Yarn crashes with local packages references in the package.jsons (`"timebrush": "file:./timebrush`)